tests/Unit/Http/Action/Security/LogInActionTest.php

```diff
@@ -109,10 +110,29 @@ public function rendersLogInFormIfAuthenticationFailed()
         $this->assertSame($content, $response->getContent());
     }
 
+    /**
+     * Data provider for redirectsToDashboardIfAuthenticationSucceeded
+     *
+     * @return array
+     */
+    public function userProvider(): array
+    {
+        return [ // admin check, reviewer check, expected redirect
+            [false, false, 'dashboard'],
+            [false, true, 'reviewer'],
+            [true, null, 'admin'],
```

Comment:
I think this should be `[true, false, 'admin']` instead (see method signature of `User::hasAccess()`.

Reply:
Negative. Because we're using an else-if on the actual impl, the is-a-reviewer check is never called. In the body of the test, the null here avoids telling Prophecy to expect a call to hasAccess for the reviewer role, because the admin role being there actually short-circuits that.